### PR TITLE
Add newrelic

### PR DESCRIPTION
**Pull request by micahsaul**
Adds New Relic config and changes how app launches in cloud.gov.

Also updates Flask to most recent version.

### CODE REVIEW

**Comment by micahsaul** [comment]
@konklone I'd love a sanity check review on this. Thanks!

File: newrelic.ini [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+# High Security Mode enforces certain security settings, and
+# prevents them from being overridden, so that no sensitive data
+# is sent to New Relic. Enabling High Security Mode means that
+# SSL is turned on, request parameters are not collected, and SQL
+# can not be sent to New Relic in its raw form. To activate High
+# Security Mode, it must be set to 'true' in this local .ini
+# configuration file AND be set to 'true' in the server-side
+# configuration in the New Relic user interface. For details, see
+# https://docs.newrelic.com/docs/subscriptions/high-security
+high_security = false
```
Comment (konklone) [inline newrelic.ini]: Any reason not to enable this?

Comment (micahsaul) [inline newrelic.ini]: Not that I know of. Happy to switch to high security if that's desired, I basically just copy-pasted the config from another one of our projects.

**Comment by micahsaul** [comment]
I'm going to go ahead and merge this now. If we want to set up high security on New Relic in the future, I'll open another PR.

**Pull request merged**